scipy/linalg/decomp.py

```diff
@@ -1283,19 +1283,19 @@ def cdf2rdf(w, v):
 
     Parameters
     ----------
-    w : array_like
-        Complex or real eigenvalues array of size (M,) for a single matrix, or
+    w : (..., M) array_like
+        Complex or real eigenvalues array of shape (M,) for a single matrix, or
         size (..., M) for multiple stacked matrices
-    v : array_like
-        Complex or real eigenvectors array of size (M, M) for a single matrix,
+    v : (..., M, M) array_like
+        Complex or real eigenvectors array of shape (M, M) for a single matrix,
```

Comment:
Is there a convention for `shape` vs `size`?

Reply:
If it's >1-D then it should always say shape I think

Reply:
Is saying `array of shape (M, M) for a single matrix,  or shape (..., M, M) for multiple stacked matrices` necessary, or is `array or stack of arrays` enough, since the type already shows the shape?

Reply:
the "array" and "stack of arrays" probably. but that's very detailed, either is fine I'd say